Can't resolve 'styled-jsx/css'

I have a brand new Next.js project set up (v4.1.4) with `styled-jsx-css-loader` configured in `next.config.js` exactly as it says in the README here and am trying to import a css file on `index.js`:

```js
import styles from '../styles/styles.css';

export default () => (
  <div>
    <style jsx>{styles}</style>
  </div>
);
```

However, getting the following error upon starting next:
```
Error in styles/styles.css
Module not found: Error: Can't resolve 'styled-jsx/css' in '...'
```
and the console error:
```
 ERROR  Failed to compile with 1 errors

This dependency was not found:

* styled-jsx/css in ./styles/styles.css

To install it, you can run: npm install --save styled-jsx/css
```

What am I doing wrong?

--- a/README.md
+++ b/README.md
@@ -65,10 +65,16 @@ module.exports = {
           {
             loader: 'emit-file-loader',
             options: {
-              name: 'dist/[path][name].[ext]',
+              name: 'dist/[path][name].[ext].js',
             },
           },
-          'babel-loader',
+          {
+            loader: 'babel-loader',
+            options: {
+              presets: ['next/babel'],
+              plugins: [require.resolve('babel-plugin-transform-es2015-modules-commonjs')]
+            }
+          },
           'styled-jsx-css-loader',
         ],
       }

